deps/csl.mk

```diff
@@ -22,7 +22,6 @@ endef
 # would get from CSL (by searching for a `GLIBCXX_3.4.X` symbol that does not exist
 # in our CSL, but would in a newer one), and default to `USE_BINARYBUILDER_CSL=0` in
 # this case.
```

Comment:
There is a leftover comment which should probably be moved to Make.inc

Reply:
@apaz-cli can you move this comment please?

Reply:
Actually, just copy it, since it makes sense in both places.

Reply:
Huh, it seems the comment I edited above is derived from the comment here, but has unique typos?! What's going on there?